app/services/audit_results.rb

```diff
@@ -149,7 +149,7 @@ def remove_db_updated_results
   #   results = [result1, result2]
   #   result1 = {response_code => msg}
   #   result2 = {response_code => msg}
-  def report_results(logger=Rails.logger)
+  def report_results(logger=Rails.logger, complete_moab=nil)
```

Comment:
I felt the most elegant way to be able to write audit results to a CompleteMoab object was to do it within the AuditResults class.  However, we only want to do this when the audit pertains to a CompleteMoab (not a Zip or Part).  Sending in the CompleteMoab instance as an arg seemed to me the most elegant way to do this.